_tests/integration/activate_test.go

```diff
@@ -51,7 +52,8 @@ func cloneDeafultStepLib() (string, error) {
 
 	cloneCmd := repo.Clone(defaultLibraryURI)
 	if out, err := cloneCmd.RunAndReturnTrimmedCombinedOutput(); err != nil {
-		if errorutil.IsExitStatusError(err) {
```

Comment:
Did you remove the errorutil package because checking for exit error is relatively easy to do it even without it?

Reply:
Package name with an underscore (`_test`) is not considered as part of the go project by the go tools. 
Because `"github.com/bitrise-io/go-utils/errorutil"` is only imported from this file, go get isn't adding it to the project's dependencies.